tests/ops/service/connectors/test_queryconfig.py

```diff
@@ -534,6 +535,11 @@ def test_generate_update_stmt_multiple_fields(
             }
         }
 
+        print(mongo_statement[1])
+        print(expected_result_1)
```

Comment:
We'll want to remove these `print`s

Reply:
see answer above, without these prints I had no way to know why and what broke